c/ntdrivers/diskperf.i.cil-2.c

```diff
@@ -2974,7 +2974,7 @@ void DiskPerfLogError(PDEVICE_OBJECT DeviceObject , ULONG UniqueId , NTSTATUS Er
     errorLogEntry->ErrorCode = ErrorCode;
     errorLogEntry->UniqueErrorValue = UniqueId;
     errorLogEntry->FinalStatus = Status;
-    memcpy(& errorLogEntry->DumpData[0], & DeviceObject, sizeof(DEVICE_OBJECT ));
+    memcpy(& errorLogEntry->DumpData[0], DeviceObject, sizeof(DEVICE_OBJECT ));
```

Comment:
DeviceObject is a pointer of type 
`typedef struct _DEVICE_OBJECT *PDEVICE_OBJECT`
And we want to copy data of size sizeof(DEVICE_OBJECT) hence DeviceObject looks more appropriate than address of its pointer &DeviceObject

Reply:
So you are agreeing with this change, are you?